types/leaflet-routing-machine/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "interface-name": [
```

Comment:
This file should only `"extends": "dtslint/dt.json"` for new packages, you can disable rules in code like `// tslint:disable-next-line interface-name`